### DISCUSSION TITLE
Add svg properties.

### DISCUSSION THREAD

**Discussion by watanabeyu** [vanilla-extract-css/vanilla-extract#1380] [Ideas]
I'm using SVG animation and I'd like support for types corresponding to SVG-specific properties like `y`.

**Comment by askoufis** [comment]
`y` (and other [geometric properties](https://css-tricks.com/svg-properties-and-css/#aa-geometry-properties)) is only supported in CSS in browser that implement the SVG 2.0 spec, which I don't think any browser fully support yet. I think Chrome implements some of it, but until there's broader support, both in browsers and the [`csstype`](https://www.npmjs.com/package/csstype) package, this probably won't be implemented anytime soon.

**Reply by askoufis** [reply]
@watanabeyu [`@vanilla-extract/css@1.17.5`](https://github.com/vanilla-extract-css/vanilla-extract/releases/tag/%40vanilla-extract%2Fcss%401.17.5) which updates `csstype` to the latest version should support SVG-specific properties.
